data/events/scripts/player.lua

```diff
@@ -86,6 +86,25 @@ function Player:onLookInShop(itemType, count)
 end
 
 function Player:onMoveItem(item, count, fromPosition, toPosition, fromCylinder, toCylinder)
+	-- Store Inbox
```

Comment:
> Tfs is a server I hope you know that? ;)
wtf? reason why i wrote that is that you said ANY SERVER (any server include tfs, or dota server and whos know how many more) so gl breaking valve's servers.

i think you still not get the point, the point of my comments, i'm repeating myself, but ok look, you agree that player walking event shouldn't stop working because of bad stepin/stepout event? so why store inbox should?